### PR TITLE
fix: Filtered Policy Support

### PR DESCRIPTION
**Pull request by yyellowsun**
Fix: https://github.com/pycasbin/sqlalchemy-adapter/issues/14

### CODE REVIEW

**Comment by yyellowsun** [comment]
@techoner  please help review the code.

**Comment by hsluoyz** [comment]
@yyellowsun plz fix CI.

**Comment by hsluoyz** [comment]
@yyellowsun plz fix the code coverage drop: 

image

File: casbin_sqlalchemy_adapter/adapter.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+from sqlalchemy import create_engine, and_, or_
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import sessionmaker
 
 Base = declarative_base()
 
 
 class CasbinRule(Base):
-    __tablename__ = 'casbin_rule'
+    __tablename__ = "casbin_rule"
```
Comment (techoner) [inline casbin_sqlalchemy_adapter/adapter.py:12]: Where there is no need to escape, please use single quotation marks (`' '`), same everywhere else !

File: tests/test_adapter.py [hunk 2]
```diff
@@ -125,3 +126,16 @@ def test_repr(self):
...
+
+    def test_loads_filtered_policy(self):
+        e= get_enforcer()
+
+        self.assertTrue(e.enforce("alice", "data1", "read"))
+        self.assertTrue(e.enforce("bob", "data2", "write"))
+        self.assertTrue(e.has_policy(['data2_admin', 'data2','read']))
+        self.assertTrue(e.has_policy(['data2_admin', 'data2','write']))
+        e.load_filtered_policy(PolicyFilter(p=(Filter(v0="alice")), g=(Filter(v0=" data1_admin"))))
+        self.assertTrue(e.enforce("alice", "data1", "read"))
```
Comment (techoner) [inline tests/test_adapter.py:138]: I think arrays should be supported in Filter, refer to [Gorm Adapter](https://github.com/casbin/gorm-adapter/blob/master/adapter_test.go#L180).

**Review by techoner — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
@yyellowsun plz revise.

**Comment by yyellowsun** [comment]
@techoner please help to review again.I don't know if it's ok,maybe something wrong in the 'g' ptype.
And I add some test examples,but coverage still droped.

**Pull request merged**